generic/tests/vlan.py

```diff
@@ -155,6 +155,23 @@ def flood_ping(src, dst):
         vm_.verify_alive()
 
     for vm_index, vm in enumerate(vms):
+        if params["os_type"] == "windows":
+            session = vm.wait_for_serial_login(timeout=login_timeout)
+            win_vlan_id = int(params.get("win_vlan_id", 900))
+            set_vlan_cmd = params["set_vlan_cmd"]
+            nicid = utils_net.get_windows_nic_attribute(session=session,
+                                                        key="netenabled", value=True,
+                                                        target="netconnectionID")
+            session.cmd(set_vlan_cmd % nicid, timeout=240)
+            time.sleep(10)
+            ifname.append(nicid)
+            dev_mac = vm.virtnet[0].mac
+            vm_ip.append(utils_net.get_guest_ip_addr(session, dev_mac,
+                                                     os_type="windows",
+                                                     linklocal=True))
```

Comment:
@sitoliu Could it be possible to print the vm ip address in the debug log? It can help to analyze test case if any issue exists.

Reply:
@pezhang Thanks, updated it.